src/Makefile.am

```diff
@@ -346,8 +348,10 @@ libbitcoin_node_a_SOURCES = \
   blockfilter.cpp \
   chain.cpp \
   consensus/tx_verify.cpp \
+  connection_types.cpp \
   dbwrapper.cpp \
   deploymentstatus.cpp \
+  eviction.cpp \
```

Comment:
nit: I wonder if the new files should be placed in `node`, as the lib is called `libbitcoin_node_a_SOURCES`

Reply:
Seems reasonable. Same for `connection_types.cpp|h`.

Reply:
Done.